docs/examples.md

```diff
@@ -0,0 +1,145 @@
+- replace any dependencies in the downstream project on the upstream's models with a cross-project `ref`
+![refs to customers and stores replaced with cross-project ref](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/33de63e1-0579-4ac0-9ff4-22099d701b99)
+
+## Connect multiple dbt projects
+
+Let's look at a slightly modified version of the example we've been working with. Instead of a single dbt project, let's imagine you're starting with two separate dbt projects connected via the "source hack":
+- project A contains the following models
+![project A's dag of models](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/75771c9c-1fa4-4cc5-b9b9-380f39091031)
+- project B contains the following models
+![project B's dag of models](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/a94657e5-c9bc-4b8b-ada5-63887bfd0ba3)
+
+We call this type of multi-project configuration the "source hack" because there are models generated by project A (`stores` and `customers`) that are defined as sources in project B. 
+
+Let's say we want to connect these two projects using model governance best practices and cross project `ref`s. 
+
+You can run the following command:
+```bash
+dbt-meshify connect --project-paths path/to/project_a path/to/project_b
+```
+
+This will make the upstream project a dependency for the downstream project, configure the "edge" models to be `public` and contracted, and replace all dependencies in the downstream project on the upstreams's models with cross-project `ref`s:
+- add a `dependencies.yml` to the _downstream_ project (in our case, project B is downstream of project A because the `transactions` model depends on some of the models that are generated by project A - `stores` and `customers`)
+TO DO: ADD SCREENSHOT ONCE BUG IS FIXED
+- add `access: public` to all models in the upstream project that are referenced by models in the downstream project
+![customers access set to public](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/9e110ca4-40c5-4013-ab89-773b59638320)
+- for all `public` models:
+    - add a `contract` config and set `enforced: true`
+    ![yml file updated with added contract config for stores model](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/800fc871-ce56-4e80-b746-8bd84aa05574)
+    - add every column's `name` and `data_type` if not already defined
+    ![yml file updated with added column names and data_types for stores model](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/48d41e17-0ad3-4a31-863b-1a8646d1d7c9)
+- replace any dependencies in the downstream project on the upstream's models with a cross-project `ref`
+![customers and stores sources replaced with cross-project ref](https://github.com/dave-connors-3/mega-corp-big-co-inc/assets/53586774/24d72b99-fbf1-489d-bda8-ccaea267981b)
+- remove unnecessary sources
```

Comment:
did you want to add a screenshot here?

Reply:
yep! i think there was a bug related to deleting the entire yml file so i hadn't added one yet, but will do!